test/integration/good/code-gen/cpp.expected

```diff
@@ -176,30 +174,29 @@ class eight_schools_ncp_model : public model_base_crtp<eight_schools_ncp_model>
         current_statement__ = 2;
         tau = lb_constrain(tau, 0);
       }
-      std::vector<local_scalar_t__> theta_tilde;
-      theta_tilde = std::vector<local_scalar_t__>(J, 0);
+      Eigen::Matrix<local_scalar_t__, -1, 1> theta_tilde;
+      theta_tilde = Eigen::Matrix<local_scalar_t__, -1, 1>(J);
       
       current_statement__ = 3;
-      for (size_t sym1__ = 1; sym1__ <= J; ++sym1__) {
-        current_statement__ = 3;
-        assign(theta_tilde, cons_list(index_uni(sym1__), nil_index_list()), in__.scalar(
-               ), "assigning variable theta_tilde");}
-      std::vector<local_scalar_t__> theta;
-      theta = std::vector<local_scalar_t__>(J, 0);
+      theta_tilde = in__.vector(J);
+      Eigen::Matrix<local_scalar_t__, -1, 1> theta;
+      theta = Eigen::Matrix<local_scalar_t__, -1, 1>(J);
       
-      current_statement__ = 6;
-      for (size_t j = 1; j <= J; ++j) {
-        current_statement__ = 5;
-        assign(theta, cons_list(index_uni(j), nil_index_list()), (mu + (tau * theta_tilde[(j - 1)])), "assigning variable theta");
-      }
+      current_statement__ = 4;
+      for (size_t sym1__ = 1; sym1__ <= J; ++sym1__) {
+        current_statement__ = 4;
+        assign(theta, cons_list(index_uni(sym1__), nil_index_list()), std::numeric_limits<double>::quiet_NaN(
+               ), "assigning variable theta");}
+      current_statement__ = 4;
+      assign(theta, nil_index_list(), add(mu, multiply(tau, theta_tilde)), "assigning variable theta");
       {
-        current_statement__ = 7;
+        current_statement__ = 5;
         lp_accum__.add(normal_log<propto__>(mu, 0, 5));
-        current_statement__ = 8;
-        lp_accum__.add(cauchy_log<propto__>(tau, 0, 5));
-        current_statement__ = 9;
+        current_statement__ = 6;
+        lp_accum__.add(normal_log<propto__>(tau, 0, 5));
+        current_statement__ = 7;
         lp_accum__.add(normal_log<propto__>(theta_tilde, 0, 1));
-        current_statement__ = 10;
+        current_statement__ = 8;
```

Comment:
Are these changes necessitated by the PR or unrelated?

Reply:
The model had to change to fit properly.